mlrun/api/crud/secrets.py

```diff
@@ -531,3 +533,22 @@ def _is_key_map_project_secret_key(self, key: str) -> bool:
     @staticmethod
     def _generate_uuid() -> str:
         return str(uuid.uuid4())
+
+
+def get_project_secret_provider(project: str) -> typing.Callable:
+    """Implement secret provider for handle the related project secret on the API side.
+
+    :param project: Project name.
+
+    :return: A secret provider function.
+    """
+
+    def _secret_provider(key: str):
```

Comment:
why underscoring? it will make function anonymous, use public function signature